engine/classes/ElggFile.php

```diff
@@ -294,12 +294,13 @@ public function close() {
 	/**
 	 * Delete this file.
 	 *
+	 * @param bool $follow_symlinks If true, will also delete the target file if the current file is a symlink
 	 * @return bool
 	 */
-	public function delete() {
+	public function delete($follow_symlinks = true) {
```

Comment:
Nevermind. Subclass signatures don't have to match if we're just adding param(s).